kolibri/deployment/default/settings/base.py

```diff
@@ -145,7 +143,7 @@
     ('es-es', 'Español'),
 ]
 
-LANGUAGE_CODE = 'en-us'
+LANGUAGE_CODE = conf.config.get("LANGUAGE_CODE") or "en-us"
```

Comment:
@rtibbles yup. It doesn't activate it if it's not in the list of allowed languages.